I'm currently a Senior Principal Scientist and Group Leader in the gRED Computational Sciences Hub at Genentech.
My group focuses on developing robust and modular software tools for implementing computational methods applied to large genomics data. 

Before joining Genentech in 2017, I first got my PhD degree in Biostatistics at Johns Hopkins University under the supervision of [Dr. Kasper Hansen](https://www.hansenlab.org), and then pursued a postdoctoral degree in Neuroinformatics at the University of Pennsylvania with [Dr. Taki Shinohara](https://www.dbeicoe.med.upenn.edu/pennsive).

## [Bioconductor](https://bioconductor.org) contributions

### As maintainer

|Package|BioC-release|BioC-devel|Description
|---|---|---|---|
|[crisprVerse](https://github.com/crisprVerse/crisprVerse)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprVerse.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprVerse/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprVerse.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprVerse/)|Easily install of the crisprVerse ecosystem|
|[crisprDesign](https://github.com/crisprVerse/crisprDesign)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprDesign.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprDesign/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprDesign.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprDesign/)|Core gRNA design package across nucleases and applications|
|[crisprBase](https://github.com/crisprVerse/crisprBase)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprBase.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprBase/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprBase.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprBase/)|Base functions and classes for CRISPR gRNA design|
|[crisprBowtie](https://github.com/crisprVerse/crisprBowtie)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprBowtie.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprBowtie/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprBowtie.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprBowtie/)|Alignment of gRNA spacer sequences using bowtie|
|[crisprBwa](https://github.com/crisprVerse/crisprBwa)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprBwa.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprBwa/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprBwa.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprBwa/)|Alignment of gRNA spacer sequences using BWA|
|[crisprScore](https://github.com/crisprVerse/crisprScore)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprScore.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprScore/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprScore.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprScore/)|On-target and off-target scoring for CRISPR gRNAs|
|[crisprScoreData](https://github.com/crisprVerse/crisprScoreData)|[![Release OK](https://bioconductor.org/shields/build/release/data-experiment/crisprScoreData.svg)](http://bioconductor.org/checkResults/release/data-experiment-LATEST/crisprScoreData/)|[![Devel OK](https://bioconductor.org/shields/build/devel/data-experiment/crisprScoreData.svg)](http://bioconductor.org/checkResults/devel/data-experiment-LATEST/crisprScoreData/)|Pre-trained models for the crisprScore package|
|[crisprViz](https://github.com/crisprVerse/crisprViz)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprViz.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprViz/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprViz.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprViz/)|Visualization of CRISPR gRNAs using genomic tracks |
|[crisprShiny](https://github.com/crisprVerse/crisprShiny)| [![Release OK](https://bioconductor.org/shields/build/release/bioc/crisprShiny.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/crisprShiny/)| [![Devel OK](https://bioconductor.org/shields/build/devel/bioc/crisprShiny.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/crisprShiny/)| Shiny interface for CRISPR gRNAs 
|[Rbwa](https://github.com/crisprVerse/Rbwa)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/Rbwa.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/Rbwa/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/Rbwa.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/Rbwa/)|R wrapper for BWA-backtrack and BWA-MEM aligners|
|[ARRmNormalization](https://github.com/Jfortin1/ARRmNormalization)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/ARRmNormalization.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/ARRmNormalization/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/ARRmNormalization.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/ARRmNormalization/)|Normalization for methylation arrays|
|[shinyMethyl](https://github.com/Jfortin1/shinyMethyl)|[![Release OK](https://bioconductor.org/shields/build/release/bioc/shinyMethyl.svg)](http://bioconductor.org/checkResults/release/bioc-LATEST/shinyMethyl/)|[![Devel OK](https://bioconductor.org/shields/build/devel/bioc/shinyMethyl.svg)](http://bioconductor.org/checkResults/devel/bioc-LATEST/shinyMethyl/)|Interactive visualization for Illumina methylation arrays|



<!--
**Jfortin1/Jfortin1** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
